server/src/importMatch.ts

```diff
@@ -6,6 +6,15 @@ export function getUriMatch(uri: string, document: TextDocument, shorthands: Map
     return isAbsolute(link) ? resolve(document.uri, link) : toAbsolutePath(document, link);
 }
 
+export function getDiMatch(uri: string): string | undefined {
+    const declaredImportMatch = /(?:module:)(.*)/;
```

Comment:
Are you using this to match keys of the YAML dictionary or lines in the file? In the latter case, it should be as below, or you'll include the colon in the declared module's name.

```suggestion
    const declaredImportMatch = /(?:module:)(.*):/;
```

Reply:
nvm, found the answer